src/Customer/CustomerProfileEdit.js

```diff
@@ -84,24 +82,13 @@ class CustomerProfileEdit extends React.Component {
   }
 
   render() {
-    if (this.state.completed) {
-      return (
-        <Redirect
-          push
-          to={{
-            pathname: `/Customer/${this.props.id}`,
-          }}
-        />
-      );
```

Comment:
why are these removed? Isn't this make the save after edit staying the form page instead of going back to the customer detail page?

Reply:
I used refresh function to reload all data in same page to check the data is changed successfully